sdk/keyvault/azkeys/models.go

```diff
@@ -306,16 +325,33 @@ type KeyReleasePolicy struct {
 	ContentType *string `json:"contentType,omitempty"`
 
 	// Blob encoding the policy rules under which the key can be released.
-	Data []byte `json:"data,omitempty"`
+	EncodedPolicy []byte `json:"data,omitempty"`
```

Comment:
This needs to be base64url-encoded, as does any other `[]byte` in the API (that we encoded, anyway; PEM input will still be base64-encoded but we don't mutate that). The REST model is therefore a string.

Reply:
This does get base64-URL encoded before sending over the wire, see [here](https://github.com/Azure/azure-sdk-for-go/blob/0f8d8fdf242f8905f8396f34ba6cdc1d9b182d94/sdk/keyvault/azkeys/internal/generated/models.go#L895).